policy/release/lib/attestations.rego

```diff
@@ -51,31 +50,34 @@ pipelinerun_attestations := att if {
 
 pipelinerun_slsa_provenance02 := [statement |
 	some att in input.attestations
-	statement := lib.statement(att)
-	statement.predicate.buildType in pipelinerun_att_build_types
+	statement := att
+	statement.statement.predicate.buildType in pipelinerun_att_build_types
```

Comment:
Hmm. Can we think of a better name for the `statement` variable? `statement.statement.` makes me think there's a bug (I don't think there is one, it just looks odd).

Maybe?

```rego
pipelinerun_slsa_provenance02 := [att |
  some att in input.attestatons
  att.statement.predicate.buildType in pipelinerun_att_build_types
]
```

Similar comment for `pipelinerun_slsa_provenance_v1` and `taskrun_attestations`.

Reply:
Whoops - I meant to fix that, but my testing adventure must have pushed it out of my mind. All are fixed.